### PR TITLE
[Storybook v8] Add file for Participant List

### PR DESCRIPTION
**Pull request by AmyL219**
# What
Screenshot 2024-07-04 at 3 51 13 PM
Screenshot 2024-07-04 at 3 51 22 PM

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook8/stories/Components/ParticipantList/ParticipantList.story.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      isScreenSharing: p.isScreenSharing,
+      isRemovable: true
+    };
+  });
+
+  const myUserId = mockParticipants[mockParticipants.length - 1].userId;
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const onParticipantRemove = (_userId: string): void => {
+    // Do something when remove a participant from list
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Components/ParticipantList/ParticipantList.story.tsx:26]: Suggestion: In the image overlay we display an alert pop up, maybe we can do something like that so no need to force the eslint disable

**Review by JoshuaLai — APPROVED** [review]
Just a few nitpick suggestions

File: packages/storybook8/stories/Components/ParticipantList/Docs.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { ParticipantList } from '@azure/communication-react';
+import { Title, Heading, Description, Canvas, Props, Meta } from '@storybook/addon-docs';
```
[the remark this answers is not in the document]
Comment (prabhjot-msft) [reply packages/storybook8/stories/Components/ParticipantList/Docs.mdx:2]: aren't we using `@storybook/blocks` for these imports and not `@storybook/addon-docs`

File: packages/storybook8/stories/Components/ParticipantList/__snapshots__/ParticipantList.stories.storyshot
Comment (prabhjot-msft) [inline packages/storybook8/stories/Components/ParticipantList/__snapshots__/ParticipantList.stories.storyshot:1]: storyshots are deprecated, we should be removing this file

File: packages/storybook8/stories/Components/ParticipantList/Docs.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (AmyL219) [reply packages/storybook8/stories/Components/ParticipantList/Docs.mdx:2]: Both `blocks` and `addon-docs` work

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**